app/src/main/java/org/akvo/flow/ui/view/CascadeQuestionView.java

```diff
@@ -239,39 +264,39 @@ public void rehydrate(QuestionResponse resp) {
             }
 
             if (valuePosition == POSITION_NONE || spinnerValues.isEmpty()) {
```

Comment:
I noticed that you replaced everything related to spinners with cascade levels.  what about the `spinnerValues`?  Does that need to be converted as well?